styles/zesty/layout.s2

```diff
@@ -753,13 +781,21 @@ h3 {
 }
 .userpic {
     margin-bottom: 5px;
+    width: 80%;
```

Comment:
This would need to go on `.userpic img` as userpics have their width and height hardcoded, and I feel like there was a reason we don't do that??? But of course I can't remember it. That said this is Zesty so I'm down for just putting the resize on the image itself and seeing what happens.

Reply:
If you're willing to let me fuck around and find out with zesty I'm down for it, haha. I went back and checked and tweaked a couple things with userpic display on mobile (and also double-checked that it looks right on my phone) and also realized that we hadn't added the meta tags so I did that also.

Reply:
Yeah I'm willing to YOLO it up with Zesty, and it looks good now even with non-square icons for me.